### DISCUSSION TITLE
Fail to build 3.4.7

### DISCUSSION THREAD

**Discussion by antonioleal** [pawelsalawa/sqlitestudio#5143] [General]
### Details
Fail to build.
Version 3.4.4 and 3.4.6 build ok :-(

### Steps to reproduce

1. antonio@Coral:~/Downloads/sqlitestudio-3.4.7/output/build$ qmake ../../SQLiteStudio3
2. antonio@Coral:~/Downloads/sqlitestudio-3.4.7/output/build$ make -j 10
3. got this one:
_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I../../../SQLiteStudio3/sqlitestudio -I. -I../../../SQLiteStudio3/coreSQLiteStudio -I../../../SQLiteStudio3/guiSQLiteStudio -I../guiSQLiteStudio -I/usr/include/qt5 -I/usr/include/qt5/QtWidgets -I/usr/include/qt5/QtGui -I/usr/include/qt5/QtNetwork -I/usr/include/qt5/QtCore -I. -I/usr/lib64/qt5/mkspecs/linux-g++ -o ./qrc_qmake_qmake_qm_files.o qrc_qmake_qmake_qm_files.cpp
g++ -c -pipe -pedantic -O2 -O2 -fPIC -std=gnu++1z -Wall -Wextra -D_REENTRANT -fPIC -DPROJECT_MODULE_NAME=sqlitestudio -DQAPPLICATION_CLASS=QApplication -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I../../../SQLiteStudio3/sqlitestudio -I. -I../../../SQLiteStudio3/coreSQLiteStudio -I../../../SQLiteStudio3/guiSQLiteStudio -I../guiSQLiteStudio -I/usr/include/qt5 -I/usr/include/qt5/QtWidgets -I/usr/include/qt5/QtGui -I/usr/include/qt5/QtNetwork -I/usr/include/qt5/QtCore -I. -I/usr/lib64/qt5/mkspecs/linux-g++ -o ./moc_singleapplication.o moc_singleapplication.cpp
g++ -c -pipe -pedantic -O2 -O2 -fPIC -std=gnu++1z -Wall -Wextra -D_REENTRANT -fPIC -DPROJECT_MODULE_NAME=sqlitestudio -DQAPPLICATION_CLASS=QApplication -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -I../../../SQLiteStudio3/sqlitestudio -I. -I../../../SQLiteStudio3/coreSQLiteStudio -I../../../SQLiteStudio3/guiSQLiteStudio -I../guiSQLiteStudio -I/usr/include/qt5 -I/usr/include/qt5/QtWidgets -I/usr/include/qt5/QtGui -I/usr/include/qt5/QtNetwork -I/usr/include/qt5/QtCore -I. -I/usr/lib64/qt5/mkspecs/linux-g++ -o ./moc_singleapplication_p.o moc_singleapplication_p.cpp
../../../SQLiteStudio3/sqlitestudio/singleapplication/singleapplication.cpp: In member function ‘void SingleApplicationPrivate::genBlockServerName(int)’:
../../../SQLiteStudio3/sqlitestudio/singleapplication/singleapplication.cpp:113:22: warning: ‘void QProcess::start(const QString&, QIODevice::OpenMode)’ is deprecated: Use QProcess::start(const QString &program, const QStringList &arguments,OpenMode mode = ReadWrite) instead [-Wdeprecated-declarations]
  113 |         process.start( "whoami" );
      |         ~~~~~~~~~~~~~^~~~~~~~~~~~
In file included from /usr/include/qt5/QtCore/QProcess:1,
                 from ../../../SQLiteStudio3/sqlitestudio/singleapplication/singleapplication.cpp:26:
/usr/include/qt5/QtCore/qprocess.h:168:10: note: declared here
  168 |     void start(const QString &command, OpenMode mode = ReadWrite);
      |          ^~~~~
g++ -Wl,-O1 -L/usr/lib64 -Wl,-rpath,/usr/lib64 -o ../../SQLiteStudio/sqlitestudio ./main.o ./singleapplication.o ./qrc_sqlitestudio.o ./qrc_qmake_qmake_qm_files.o ./moc_singleapplication.o ./moc_singleapplication_p.o   -L/mnt/data/antonio/Downloads/sqlitestudio-3.4.7/SQLiteStudio3/../output/SQLiteStudio -lcoreSQLiteStudio -lguiSQLiteStudio /usr/lib64/libQt5Widgets.so /usr/lib64/libQt5Gui.so /usr/lib64/libQt5Network.so /usr/lib64/libQt5Core.so -lGL -lpthread   
/usr/bin/ld: /mnt/data/antonio/Downloads/sqlitestudio-3.4.7/SQLiteStudio3/../output/SQLiteStudio/libcoreSQLiteStudio.so: undefined reference to `sqlite3_is_interrupted'
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:445: ../../SQLiteStudio/sqlitestudio] Error 1
make[1]: Leaving directory '/mnt/data/antonio/Downloads/sqlitestudio-3.4.7/output/build/sqlitestudio'
make: *** [Makefile:125: sub-sqlitestudio-make_first] Error 2

### Operating system
Slackware64 15.0

### SQLiteStudio version
Version 3.4.7

**Comment by pawelsalawa** [comment]
You need SQLite at least 3.41.0 to compile.
